packages/xo-web/src/common/utils.js

```diff
@@ -481,26 +481,6 @@ export const isXosanPack = ({ name }) => startsWith(name, 'XOSAN')
 
 // ===================================================================
 
-export const getCoresPerSocketPossibilities = (maxCoresPerSocket, vCPUs) => {
-  // According to : https://www.citrix.com/blogs/2014/03/11/citrix-xenserver-setting-more-than-one-vcpu-per-vm-to-improve-application-performance-and-server-consolidation-e-g-for-cad3-d-graphical-applications/
-  const maxVCPUs = 16
```

Comment:
Why don't you need it?

Reply:
Because it's not correct. [The max vCPUS for HVM is 128 and for PV is 512](https://wiki.xen.org/wiki/Xen_Project_Release_Features). 
In this PR, i used the VM max VCPUs directly.

Reply:
IMHO we should know the relationship between the number of cores per socket and nVcpus. 
The user doesn't know the number of cores per sockets or the number of sockets to solve this error.

Reply:
I updated the implementation, the property name is wrong is not `maxVCPUS` but `maxSockets`.